architecture/aws/observability.md

```diff
@@ -1,65 +1,56 @@
 ---
 title: AWS observability services
 summary:
-reviewed: 2024-03-14
+reviewed: 2024-03-28
 callsToAction: ['solution-architect', 'poc-help']
```

Comment:
As a general rule, Nisha and I were thinking the `solution-architect` call to action makes sense for the technology choices and the `ADSD` one for the concepts